headers/types/common/common.h

```diff
@@ -13,6 +13,23 @@ struct iovec {
 };
 ASSERT_SIZE(struct iovec, 8);
 
+struct rect16_xywh {
+    int16_t x;
+    int16_t y;
+    int16_t w;
+    int16_t h;
+};
+ASSERT_SIZE(struct rect16_xywh, 8);
+
+// The fact the 4th byte is never used hints to the fact the struct is packed
+struct rgbx {
+    uint8_t r;
+    uint8_t b;
+    uint8_t g;
+    uint8_t x;
```

Comment:
That'd be smart, although I mostly called it RGBX because that's how many of the existing tech documents call it. There's also a chance that the value might be used in some scenarios (most of the time, x = 0x80), although for WTE I'm pretty certain it's not used at all

Reply:
Oh okay, I assumed it would always be 0 or something. If there's already a convention about it we can just leave it as `rgbx` then.